src/runners/JasmineComplianceRunner.ts

```diff
@@ -0,0 +1,74 @@
+import { APIGatewayEvent, Callback, Context, Handler } from 'aws-lambda';
+
+const Jasmine = require('jasmine')
+const TSConsoleReporter = require('jasmine-ts-console-reporter');
+
+export const lambdaNotificationReporter = (callback: Callback) => {
+  return {
+    results : [],
+    testErrors: [],
+    passedExpectations : [],
+    jasmineDone: function (result) {
+      if (result.overallStatus === 'failed') {
+        let failedTests = this.testErrors.map(x => x.fullName).join(',');
```

Comment:
Minor: join with ', ' (with a space). I forgot this when I originally wrote it.

Reply:
ok - will fix.